### ISSUE TITLE
 f

### ISSUE THREAD

**Issue by daniellybx** [nsbatra/Epi_R_handbook#124]

**Comment by nsbatra** [comment]
Closing

**Comment by nsbatra** [comment]
Hi Danielly,
Thank you for your review! I agree it is important to offer base R options
for exactly that reason (not able to download other packages). These
updates should be on the website later tonight.

I have added the NULL option to remove a data frame column.

The duplicated() command is described in the page on de-duplication, so I
would rather not repeat that in this cleaning page. Here in the cleaning
page I only show one approach to de-duplication. If the reader wants more
they can click the link to the De-duplicaton page. I have made the text
more clear to direct the reader to this other page.

In regards to creating a column with base R syntax, there actually is a
small section on this at the bottom of the "Column creation and
transformation" section. I will update this section to include the BMI
example.

For is.na() - this is described in detail in the "Missing data" page (a
link is provided from the Cleaning page). But I have added it again to the
cleaning page as you suggest.

Thanks for all your reviews, they are very helpful!
